### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Storage Container: Search icon present beside the search edit field looks as ‘Run’ button 

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3561]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS3.2.4; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS3.2.4

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Open Eclipse and install the Scala plug-in 
4) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5) Tab till Windows menu option-->Show View-->Other-->Expand Azure-->Azure Explorer pane press OK
6) In Azure Explorer section, tab till HDInsight and expand it. Expand the cluster name
7) Tab till the storage container and press enter
8) In Storage Container Section, tab till search icon and verify how the icon looks like

**Actual Result:** 
Search icon present beside the search edit field looks as ‘Run’ button 

**Expected Result:** 
All the controls in the application must have consistent identification and Search icon should be differentiated and should not be like ‘Run’ icon

**User Impact:** 
Users will find difficulty in understanding the purpose if two controls are designed similarly

**MAS Reference:** 
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={2a3e26b6-fe3e-4205-8370-ee24fa1764ac}

[Bug_10_consistent identification.pptx](https://github.com/microsoft/azure-tools-for-java/files/3635610/Bug_10_consistent.identification.pptx)

**Comment by t-rufang** [comment]
Note: The icon is the same as the `Run` button, but the actual meaning of the icon is `Search`.
![image](https://user-images.githubusercontent.com/32627233/65827973-60fc6100-e2c8-11e9-8149-ccbf8b3bb335.png)

**Comment by Flanker32** [comment]
@Tanushagottipati We have removed storage explorer in IDE toolkit, could you please help us verify and close the issue? Here is the [build](https://dev.azure.com/mseng/VSJava/_build/results?buildId=11079936)

**Comment by Tanushagottipati** [comment]
As mentioned storage accounts tab cannot be opened in the eclipse toolkit. As storage explorer has been deprecated, closing this bug and attached screenshot for reference.
Environment:
Eclipse IDE for Enterprise Java Developers.
Version: 2019-09 R (4.13.0)
Build id: 20190917-1200
Solution Installed: The Azure Toolkit for Eclipse 3.20.0
![Screenshot (709)](https://user-images.githubusercontent.com/41109135/70886831-7c8f2200-2002-11ea-8f2b-4a1fab79bcaa.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749673
